### ISSUE TITLE
Unable to send SMS via python

### ISSUE THREAD

**Issue by Nexite** [Seeed-Studio/ublox_lara_r2_pi_hat#1]
I'm trying to send a message with python user sendAT and it gives the error ERROR

`
from ublox_lara_r2 import *

u = Ublox_lara_r2()
u.initialize()
u.reset_power()
 
u.debug = True

u.sendAT("AT+CMGF=1\n")
u.sendAT('AT+CMGS="##########"\rThis is a message\x1a')
`

**Comment by lanselambor** [comment]
@TheDestoyer You need to separate phone number and message into two commands, the first command contains phone number would return ">", only after that return code, message can be put in.

**Comment by KillingJacky** [comment]
Will close this issue, please open a new issue if you have further questions.
